pkg/api/types.go

```diff
@@ -1232,6 +1232,9 @@ func (p *Properties) ShouldEnableAzureCloudAddon(addonName string) bool {
 	if !to.Bool(o.KubernetesConfig.UseCloudControllerManager) {
 		return false
 	}
+	if addonName == common.AzureDiskCSIDriverAddonName && p.IsAzureStackCloud() {
```

Comment:
Update the method documentation please:

`// For Azure Stack Hub clusters, ...`

Reply:
added comment